pages/about-us/press/en.md

```diff
@@ -0,0 +1,69 @@
+* [[How to protest against Trump in his expanded surveillance state->https://www.newscientist.com/article/mg23331092-700-whats-the-worst-that-could-happen]]
+
+### 2016
+
+* [[Surveillance Self-Defense Against the Trump Administration->https://theintercept.com/2016/11/12/surveillance-self-defense-against-the-trump-administration/]]
+* [[Something Happened to Activist Email Provider Riseup, but It Hasn’t Been Compromised->https://theintercept.com/2016/11/29/something-happened-to-activist-email-provider-riseup-but-it-hasnt-been-compromised/]]
+* [[Riseup email service sparks concern after warrant canary 'expires'->https://www.zdnet.com/article/encrypted-email-provider-riseup-misses-warrant-canary-deadline/]]
+* [[Internet Service Providers You Can Trust->https://indypendent.org/2016/08/15/internet-service-providers-you-can-trust]]
+* [[Report: Impacts of surveillance on contemporary British activism->https://www.opendemocracy.net/uk/gilbert-ramsay/report-impacts-of-surveillance-on-contemporary-british-activism]]
+
+### 2014
+
+* [[ISPs take legal action against GCHQ -> https://www.bbc.com/news/technology-28106815]]
+
+### 2013
+
+* [Ars Technica, Europe Won't Save You, Why Email is Probably Safer in the US](https://arstechnica.com/tech-policy/2013/10/europe-wont-save-you-why-e-mail-is-probably-safer-in-the-us/)
+
+### 2012
+
+* [N+1 Magazine - Leave Your Cellphone at Home](https://nplusonemag.com/leave-your-cellphone-at-home)
+
+> "For email, using Riseup.net is good news. The solutions they offer are integrated with Tor as much as possible. They’re badass."
+
+* [New York Times - How to Muddy Your Tracks on the Internet](https://www.nytimes.com/2012/05/03/technology/personaltech/how-to-muddy-your-tracks-on-the-internet.html)
+
+> "If you do not want the content of your e-mail messages examined or analyzed at all, you may want to consider lesser-known free services like HushMail, RiseUp and Zoho, which promote no-snooping policies."
+
+Media accounts of the Riseup server seizure:
+
+1. [Forbes - FBI Seizes Activists' Anonymizing Server In Probe Of Pittsburgh's Bomb Threats](https://www.forbes.com/sites/andygreenberg/2012/04/19/fbi-seizes-activists-anonymizing-server-in-probe-of-pittsburghs-bomb-threats/)
+1. [The Register - FBI seizes Mixmaster servers](https://www.theregister.co.uk/2012/04/19/mixmaster_servers_seized/)
+1. [PC World - FBI Seizes Anonymizing Server in Bomb Threat Probe](https://www.pcworld.com/businesscenter/article/254128/fbi_seizes_anonymizing_server_in_bomb_threat_probe.html)
+1. [Information Week - FBI Seizes Anonymizing Email Service Server](https://www.informationweek.com/news/security/government/232900643)
+1. [taz.de](https://taz.de/!91855/) (German)
+# [techdirt - FBI Seized Anonymizer Server](https://www.techdirt.com/articles/20120420/04570818576/fbi-seized-anonymizer-server.shtml)
```

Comment:
This techdirt line produces a weird format in the middle of the numbers, probably the # needs to be a 1?

Reply:
There are no such languages for that page https://github.com/riseupnet/riseup_help/tree/master/pages/about-us/press

Reply:
Yeah, it is just using the english one for those other languages, so if its fixed on that one, it will be fixed on the others.